tests/ctst/common/common.ts

```diff
@@ -8,6 +8,51 @@ import assert from 'assert';
 
 setDefaultTimeout(Constants.DEFAULT_TIMEOUT);
 
+type listingObject = {
+    Key: string,
+    VersionId: string,
+}
+
+type listingResult = {
+    Versions: listingObject[],
+    DeleteMarkers: listingObject[],
+}
+
+/**
+ * Cleans the created test bucket
+ * @param {Zenko} world world object
+ * @param {string} bucketName bucket name
+ * @returns {void}
+ */
+export async function cleanZenkoBucket(
+    world: Zenko,
+    bucketName: string,
+): Promise<void> {
+    if (!bucketName) {
+        return;
+    }
+    world.resetCommand();
+    world.addCommandParameter({ bucket: bucketName });
+    const createdObjects = world.getSaved<Map<string, string>>('createdObjects');
+    if (createdObjects !== undefined) {
+        const results = await S3.listObjectVersions(world.getCommandParameters());
+        const res = safeJsonParse(results.stdout);
+        assert(res.ok);
+        const parsedResults = res.result as listingResult;
+        const versions = parsedResults.Versions || [];
+        const deleteMarkers = parsedResults.DeleteMarkers || [];
+        await Promise.all(versions.concat(deleteMarkers).map(obj => {
+            world.addCommandParameter({ key: obj.Key });
+            world.addCommandParameter({ versionId: obj.VersionId });
```

Comment:
* does `addCommandParameter` have the semantic to replace an existing parameter with the same "key" ? Otherwise we may end up with multiple "keys" in the parameters.... (which may be ok, as aws cli to take into account the last one; but seems flimsy anyway)
* are we able to both params in a single call to `addCommandParameter`, as would be natural in nodejs? (i.e. `world.addCommandParameter({ key: obj.Key, versionId: obj.VersionId });`)

Reply:
> does addCommandParameter have the semantic to replace an existing parameter with the same "key" ?

It is replacing yes, so we won't face this issue

> are we able to both params in a single call to addCommandParameter, as would be natural in nodejs? (i.e. world.addCommandParameter({ key: obj.Key, versionId: obj.VersionId });)

We are not yet able to do that.. i created ZENKO-4708 to address that